fourth_task/src/MockGui.ts

```diff
@@ -1,13 +1,15 @@
 import { GuiInterface } from './Gui';
 import { ShipmentInterface } from './Shipment';
 
+const mockedMarks: string[] = ['Fragile', 'Do Not Leave', 'Return Receipt Requested'];
 const mockedShipment: ShipmentInterface = {
   shipmentId: 1,
   toAddress: 'Lviv, Svobody str',
   fromAdress: 'Kyiv, Bankova str',
   toZipCode: '79000',
   fromZipCode: '01001',
-  weight: 10
+  weight: 10,
+  marks: mockedMarks
```

Comment:
@dianalemen, how do you test the code?

Reply:
@AlexHladin I tested it in codepen

Reply:
@dianalemen, I mean what are you running?

Reply:
@AlexHladin if you want to run it locally from the terminal, there is no command for it. I've not configured it, because it was not the requirement.  I've tested it on the codepen. 
Just copy pasted all classes and run new Client()
Also in order to see something on the console, you have to add console.log to the ship method in the ShipmentDecorator